### DISCUSSION TITLE
AG 1.2 buffer issue

### DISCUSSION THREAD

**Discussion by jaacMusic** [apohl79/audiogridder#508] [Help]
I'm experiencing audio dropouts and buffer issues. I have configured AG plugin to use 30 Blocks buffer size and I still get audio dropouts, skips, and sync issues. These issues was not present when using 1.1 version. AGridder Monitor keeps popping up with plugins with high latency values (ms).
![image](https://user-images.githubusercontent.com/68957086/115610999-91935980-a2b7-11eb-8fdf-90180d35408e.png)

**Comment by CaBuffalo** [comment]
I'm experiencing the similar issue when I changed the buffer size of a
session. It was disappeared after change the value back to the original
number.

Try it and see if it's the case.

Andreas should be able to take a look at it next.

On Wed, Apr 21, 2021 at 12:39 PM jaacMusic ***@***.***> wrote:

> I'm experiencing audio dropouts and buffer issues. I have configured AG
> plugin to use 30 Blocks buffer size and I still get audio dropouts, skips,
> and sync issues. These issues was not present when using 1.1 version.
> AGridder Monitor keeps popping up with plugins with high latency values
> (ms).
> [image: image]
> <https://user-images.githubusercontent.com/68957086/115610999-91935980-a2b7-11eb-8fdf-90180d35408e.png>
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/508>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6C66YNESQIEPLT3SXDTJ4SWFANCNFSM43K7TYJA>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by jaacMusic** [reply]
I just realized the issue has to do with the Waves InPhase LT Stereo plugin. I use this plugin to move back and forward in ms audio recordings tracks that are not in complete synch. 
By moving the plugin to the DAW instead of running from the AG plugin solved the issue. This is something that didn't happen with version 1.1.1.

**Reply by apohl79** [reply]
If you can provide a set of plugin and server logs from 1.1 and 1.2 for comparison I can take a look.
